release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -291,6 +291,34 @@ Currently, it is not supported to use Active Directory Federation Services with
 
 For the full list of OIDC providers, see xref:../authentication/identity_providers/configuring-oidc-identity-provider.adoc#identity-provider-oidc-supported_configuring-oidc-identity-provider[Supported OIDC providers].
 
+[id="ocp-4-11-auth-pod-security-admission"]
+==== Pod security admission autolabeling
+
+link:https://kubernetes.io/docs/concepts/security/pod-security-admission/[Pod security admission] is now enabled on {product-title}. 
+
+If the pod security admission plug-in is activated, {product-title} preserves pod workloads despite changes to the pod security API. A cluster performs this action by monitoring administrator-labeled namespaces for changes and mapping link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security standards] to xref:../authentication/managing-security-context-constraints.adoc[security context constraints].
```

Comment:
`If the pod security admission plug-in is activated, {product-title} preserves pod workloads despite changes to the pod security API`
I don't understand this, and I think we can simply leave it out, because pod security admission is always on.

Reply:
```suggestion
Clusters now monitor administrator-labeled namespaces for changes and map link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security standards] to xref:../authentication/managing-security-context-constraints.adoc[security context constraints].
```